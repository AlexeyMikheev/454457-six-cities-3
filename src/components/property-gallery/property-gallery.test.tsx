import * as React from "react";
import {mount} from "enzyme";
import PropertyGallery from "./property-gallery";

const mocks: string[] = [
  `img/room.jpg`,
  `img/apartment-01.jpg`,
  `img/apartment-02.jpg`,
  `img/apartment-03.jpg`,
  `img/studio-01.jpg`,
  `img/apartment-01.jpg`,
];

it(`PropertyGallery snapshot`, () => {

  const div = document.createElement(`div`);
  document.body.appendChild(div);

  const propertyGalleryComponent = mount(<PropertyGallery images={mocks} />, {attachTo: div});

  expect(propertyGalleryComponent).toMatchSnapshot();
});
