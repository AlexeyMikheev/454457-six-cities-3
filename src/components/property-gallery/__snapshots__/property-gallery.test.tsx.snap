// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyGallery snapshot 1`] = `
<PropertyGallery
  images={
    Array [
      "img/room.jpg",
      "img/apartment-01.jpg",
      "img/apartment-02.jpg",
      "img/apartment-03.jpg",
      "img/studio-01.jpg",
      "img/apartment-01.jpg",
    ]
  }
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
      key="img/room.jpg0"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/room.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
      key="img/apartment-01.jpg1"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
      key="img/apartment-02.jpg2"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-02.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
      key="img/apartment-03.jpg3"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-03.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
      key="img/studio-01.jpg4"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/studio-01.jpg"
      />
    </div>
    <div
      className="property__image-wrapper"
      key="img/apartment-01.jpg5"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="img/apartment-01.jpg"
      />
    </div>
  </div>
</PropertyGallery>
`;
